### PR TITLE
Branding redesign

### PR DESCRIPTION
**Pull request by Stoppedpuma**
Redesigns banner and icons. Icons have the three variants @woelper chose. This is being opened as a PR to master instead of the redesign branch to be ready for publishing on flathub.

These assets can be downloaded [here!](https://www.figma.com/design/we5ws6DFbPjbLMKKnhOpIb/oculante-Branding-Update-2024-09-16?node-id=0-1&t=xbVxU4bGCzEhFkA4-1)

### CODE REVIEW

**Comment by Stoppedpuma** [comment]
@woelper This should be ready to merge if you're finished with your review!

**Review by woelper — CHANGES_REQUESTED** [review]
Could you please revert the avif icons back to png? The format may be better, but I like to keep things simple. I also suspect copying into a file with png extension will work, as in the `pkgbuild`.

File: PKGBUILD [hunk 1]
```diff
@@ -21,8 +21,8 @@ build() {
 package() {
     cd "$srcdir/$pkgname-$pkgver"
     install -Dm755 target/release/oculante "${pkgdir}/usr/bin/${pkgname}"
-	install -Dm644 res/oculante.png "${pkgdir}/usr/share/icons/hicolor/128x128/apps/${pkgname}.png"
+	install -Dm644 res/icons/icon.avif "${pkgdir}/usr/share/icons/hicolor/128x128/apps/${pkgname}.png"
```
[the remark this answers is not in the document]
Comment (Stoppedpuma) [reply PKGBUILD:24]: The icons are now png again

Comment (woelper) [inline PKGBUILD:24]: This needs to be changed to png please since the avif is no longer present

**Review by woelper — CHANGES_REQUESTED** [review]

**Review by woelper — APPROVED** [review]

**Pull request merged**